packages/sales-api-service/src/services/__tests__/recurring-payments.service.spec.js

```diff
@@ -259,6 +259,26 @@ describe('recurring payments service', () => {
           endDate: '2025-01-30T23:59:59.999Z'
         },
         '2025-11-12T00:00:00.000Z'
+      ],
+      [
+        "issued on 29th Feb '24, starts on 30th March '24 - next due on 28th Feb '25",
+        'hy7u8ijhyu78jhyu8iu8hjiujn',
+        {
+          startDate: '2024-03-30T00:00:00.000Z',
+          issueDate: '2024-02-29T12:38:24.123Z',
+          endDate: '2025-03-29T23:59:59.999Z'
+        },
+        '2025-02-28T00:00:00.000Z'
+      ],
+      [
+        "issued on 30th March '25 at 1am, starts at 1:30am - next due on 20th March '26",
```

Comment:
I'd suggest testing this for 12:30am 1st of April instead, as 1am on 30 March 2025 is precisely when BST starts and so technically immediately skips to 2am. Also if we test at 12:30 then we will land in the window where it's one date in BST and another date in UTC, which I think is worth testing.